packages/aspell.rb

```diff
@@ -21,7 +21,7 @@ class Aspell < Package
   })
 
   depends_on 'ruby' unless File.exists? "#{CREW_PREFIX}/bin/ruby"
-  depends_on 'ncursesw'
+  depends_on 'ncurses'
```

Comment:
If you leave the `ncursesw` dependency, you get both `ncurses` and `ncursesw`.  Any reason not to have `ncursesw`?

Reply:
the modified ncurses have both ncurses and ncursesw